src/fieldToConceptIdMapping.js

```diff
@@ -262,33 +262,39 @@ export const conceptIds = {
     },
 
     collectionLocationMapping: {
-        777644826 : 'UC-DCAM',
-        692275326 : 'Marshfield',
-        813701399 : 'Weston',
-        698283667 : 'Lake Hallie',
-        834825425 : 'HP Research Clinic',
-        574368418 : "HP Park Nicollet",
-        736183094 : 'HFH K-13 Research Clinic',
-        886364332 : 'HFH Cancer Pavilion Research Clinic',
-        706927479 : 'HFH Livonia Research Clinic',
-        322059622 : "HFH Pop-Up",
-        589224449 : 'Sioux Falls Imagenetics',
-        127626388 : "Bismarck Medical Center",
+        777644826: 'UC-DCAM',
+        692275326: 'Marshfield',
+        813701399: 'Weston',
+        698283667: 'Lake Hallie',
+        834825425: 'HP Research Clinic',
+        574368418: "HP Park Nicollet",
+        736183094: 'HFH K-13 Research Clinic',
+        886364332: 'HFH Cancer Pavilion Research Clinic',
+        706927479: 'HFH Livonia Research Clinic',
+        322059622: "HFH Pop-Up",
+        589224449: 'Sioux Falls Imagenetics',
+        127626388: "Bismarck Medical Center",
         246137578: 'Sioux Falls Sanford Center',
-        145191545 : 'Ingalls Harvey',
-        489380324 : 'River East',
-        319518299 : 'UCM Pop-Up',
-        940329442 : 'Orland Park',
-        567969985 : 'MF Pop-Up',
-        120264574 : 'South Loop',
-        691714762 : 'Rice Lake',
-        487512085 : 'Wisconsin Rapids',
-        983848564 : 'Colby Abbotsford',
-        261931804 : 'Minocqua',
-        665277300 : 'Merrill',
-        467088902 : 'Fargo South University',
-        111111111 : 'NIH/NCI',
-        807835037 : 'Other'
+        145191545: 'Ingalls Harvey',
+        489380324: 'River East',
+        319518299: 'UCM Pop-Up',
+        940329442: 'Orland Park',
+        567969985: 'MF Pop-Up',
+        120264574: 'South Loop',
+        691714762: 'Rice Lake',
+        487512085: 'Wisconsin Rapids',
+        983848564: 'Colby Abbotsford',
+        261931804: 'Minocqua',
+        665277300: 'Merrill',
+        467088902: 'Fargo South University',
+        111111111: 'NIH/NCI',
+        807835037: 'Other',
+        723351427: 'BCC- HWC, BC',
```

Comment:
Minor readability concern but were the extra indents here intentional?

Reply:
I am not sure if this Github displaying this incorrectly. My code has them aligned without the extra indents.